### PR TITLE
New package: EUSO.eParakstītājs3.0 version 1.7.2

### PR DESCRIPTION
**Pull request by Arturslv**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122908)

### CODE REVIEW

**Comment by Arturslv** [comment]
@microsoft-github-policy-service agree

File: manifests/e/EUSO/eParakstītājs3/0/1.7.2/EUSO.eParakstītājs3.0.locale.LV.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: EUSO.eParakstītājs3.0
+PackageVersion: 1.7.2
+PackageLocale: LV
+Publisher: EUSO
+PackageName: eParakstītājs 3.0
+License: eParaksts  2017 VAS "Latvijas Valsts radio un televīzijas centrs"
+ShortDescription: eParakstitajs3.0(MSI)V_1.7.2
```
Comment (Trenly) [inline manifests/e/EUSO/eParakstītājs3/0/1.7.2/EUSO.eParakstītājs3.0.locale.LV.yaml:10]: Please provide a better description of what this software does or what this software is used for

File: manifests/e/EUSO/eParakstītājs3/0/1.7.2/EUSO.eParakstītājs3.0.locale.LV.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: EUSO.eParakstītājs3.0
+PackageVersion: 1.7.2
+PackageLocale: LV
+Publisher: EUSO
+PackageName: eParakstītājs 3.0
+License: eParaksts  2017 VAS "Latvijas Valsts radio un televīzijas centrs"
```
Comment (Trenly) [inline manifests/e/EUSO/eParakstītājs3/0/1.7.2/EUSO.eParakstītājs3.0.locale.LV.yaml:9]: An eSignature is not a license. Please update to actual licence terms such as `Proprietary`, `MIT`, `GPL-3.0`, or similar

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/e/EUSO/eParakstītājs3/0/1.7.2/EUSO.eParakstītājs3.0.locale.LV.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: EUSO.eParakstītājs3.0
+PackageVersion: 1.7.2
+PackageLocale: LV
+Publisher: EUSO
+PackageName: eParakstītājs 3.0
```
Comment (stephengillie) [inline manifests/e/EUSO/eParakstītājs3/0/1.7.2/EUSO.eParakstītājs3.0.locale.LV.yaml:8]:
```suggestion
PackageName: eParakstītājs 3.0
PackageUrl: https://www.eparaksts.lv/lv/lejupielades
```

**Pull request closed without merging**